Java/src/main/java/com/gildedrose/AgedBrieItemUpdater.java

```diff
@@ -0,0 +1,29 @@
+package com.gildedrose;
+
+public class AgedBrieItemUpdater extends ItemUpdater{
+    @Override
+    void setQualityForValidItem(Item item) {
+        if (isQualityLessThanFifty(item)) {
```

Comment:
I was wondering if it wouldn't be better to have a more flexible function - one that would accept `50` as an argument. It would make your code more universal. What do you think about it?